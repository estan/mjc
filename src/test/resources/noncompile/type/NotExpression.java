class NotExpressionTest {
    public static void main(String[] args) {
        int i;
        int[] ia;
        boolean b;
        A cA;
        B cB;

        if (!i) {} // INVALID_UNARY_OP
        if (!ia) {} // INVALID_UNARY_OP
        //if (!b) {} // OK!
        if (!cA) {} // INVALID_UNARY_OP
    }
}
class A {}
class B {}
