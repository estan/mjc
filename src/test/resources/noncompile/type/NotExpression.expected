INVALID_UNARY_OP
INVALID_UNARY_OP
INVALID_UNARY_OP
